File: web3rpc/rpc-specs/code-samples/python/kaia/block/getStorageAt.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0x295a70b2de5e3953354a6a8344e616ed314d7251"
position = "0x0"
blockHash = "latest"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.get_storage_at(address, position, blockHash)

print(kaia_response)

File: web3rpc/rpc-specs/code-samples/javascript/kaia/block/getStorageAt.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const storageAddress = "0x295a70b2de5e3953354a6a8344e616ed314d7251";
  const position = "0x0";
  const blockNumberOrHash = "latest";

  provider.kaia
    .getStorageAt(
      storageAddress,
      position,
      blockNumberOrHash,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
